packages/server/lib/open_project.coffee

```diff
@@ -65,6 +66,13 @@ create = ->
           if am = options.automationMiddleware
             automation.use(am)
 
+          onBrowserClose = options.onBrowserClose
+          options.onBrowserClose = ->
+            if spec
+              preprocessor.removeFile(spec, cfg)
```

Comment:
Rather than remove individual file watchers... why wouldn't we just always call `preprocessor.close()` to close all open watchers whenever the browser is closed?

That would be easier to handle edge cases or state.

Reply:
We open a file watcher on the support file when a project is open and it stays open until the project is closed. We open a file watcher on the spec file when the browser is opened to run that spec file and this now closes that watcher when the browser is closed. We don't want to close the file watcher for the support file when the browser is closed.